README.md

```diff
@@ -2,6 +2,16 @@
 
 A fresh start for 18f.gsa.gov. Online at beta.18f.gov. We plan to use the U.S. Web Design standards as a frontend framework.
 
+## Installation
+
+Copy and paste each of the following steps to get the site up and running.
```

Comment:
I would say "Run each of the following steps to get the site up and running" or just include where to copy and paste to